manuscript/03.results.md

```diff
@@ -4,16 +4,15 @@
 
 <!-- ASM proofed. -->
 
-To illustrate the unique value of lineage measurements when analyzing heterogeneous drug responses, we plotted a series of lineages from our measurements of the breast cancer cell line AU565 treated with gemcitabine (Fig. @fig:lineage). Gemcitabine is a chemotherapy agent that disrupts DNA replication and results in extension of and apoptosis in S phase [@pmid:11504793]. While there is a striking difference in the growth dynamics of individual cells, the total number of cells at any given time is nearly identical. As a result, population-level drug response measurements cannot distinguish the differences in these two conditions. Measurements that record the history of cells (e.g., CFSE staining, Luria-Delbruck experiment) can help to distinguish these two populations but must make assumptions about the dynamics of heterogeneity [@DOI:10.1073/pnas.1715639115; @PMID:28607484]. Lineage measurements, by contrast, provide sufficiently rich measurements to quantify the specific structure of heterogeneous response. By observing individual cells, a variety of phenotypic measurements such as inter-mitotic times, cell cycle phase durations, motility, cell death and division, morphology, and protein and transcription factor levels can be characterized in parallel (CITE).
+To illustrate the unique value of lineage measurements when analyzing heterogeneous drug responses, we plotted a series of lineages from our measurements of the breast cancer cell line AU565 treated with gemcitabine (Fig. @fig:lineage). Gemcitabine is a chemotherapy agent that disrupts DNA replication and results in extension of and apoptosis in S phase [@pmid:11504793]. Comparing a few random lineages in control condition and gemcitabine-treated cells demonstrates that while there is a striking difference in the growth dynamics of individual cells, the total number of cells at any given time is nearly identical in these two conditions. As a result, population-level drug response measurements cannot distinguish the differences in these two conditions. Measurements that record the history of cells (e.g., CFSE staining, Luria-Delbruck experiment) can help to distinguish these two populations but must make assumptions about the dynamics of heterogeneity [@DOI:10.1073/pnas.1715639115; @PMID:28607484]. Lineage measurements, by contrast, provide sufficiently rich measurements to quantify the specific structure of heterogeneous response. By observing individual cells, a variety of phenotypic measurements such as inter-mitotic times, cell cycle phase durations, motility, cell death and division, morphology, and protein and transcription factor levels can be characterized in parallel [@doi:10.1126/science.abc1944]. 
```

Comment:
Does this paper reference demonstrate the many different measurements one can make in single cells? Why reference it here?

Reply:
It doesn't. I should have placed it somewhere else... I will replace it with another reference.